src/ucp/core/ucp_ep.c

```diff
@@ -3246,9 +3258,12 @@ int ucp_ep_do_keepalive(ucp_ep_h ep, ucs_time_t now)
 
     UCP_WORKER_THREAD_CS_CHECK_IS_BLOCKED(ep->worker);
 
-    ucs_assert(!(ep->flags & UCP_EP_FLAG_FAILED));
-    ucs_assert((rsc_index != UCP_NULL_RESOURCE) ||
-               (lane == ucp_ep_get_cm_lane(ep)));
+    ucs_assertv(!(ep->flags & UCP_EP_FLAG_FAILED), "ep %p flags 0x%x",
```

Comment:
"ep=%p flags=0x%x ..."
in all ucs_assertv() uses

Reply:
done